### DISCUSSION TITLE
What is special material relationships?

### DISCUSSION THREAD

**Discussion by yanboli** [bepu/bepuphysics1#29] [Q&A]
I am simulation vehicle tire on the ground without special treatment.   I notice significant tire slippage especially on slopes. I need absolute Zero slippage.  Is there a way to achieve it?

KineticFriction:   "Get of set the friction coefficient ...  and no special material relationship is defined between the colliding objects "

This sounds interesting. So, what is "special material relationship"

**Comment by RossNordby** [comment]
The material for a pair of entities is computed by the `MaterialManager.GetInteractionProperties`: https://github.com/bepu/bepuphysics1/blob/master/BEPUphysics/Materials/MaterialManager.cs#L52

It first examines the `MaterialInteractions` dictionary to see if the relationship is explicitly defined. If not, it uses the `MaterialBlender` callback. So you could either specify the desired material interaction behavior in the dictionary, or supply a custom callback.
